block/block.py

```diff
@@ -150,6 +169,13 @@ def reset_nonce(self):
     #     # calculates the size of the transactions
 
 
+def genesis_block():
+    'Mines the genesis block. (Always the same block) 0000669a5bd0d672a499608a45c24649584fb0b40c8ef6a5f9e6765caf5ae892'
+    b = Block(previous_hash='0000000000000000000000000000000000000000000000000000000000000000', transactions='test')
+    Block.target(b, 4)
+    Block.genesis_timestamp(b)
+    Block.mine(b)

```

Comment:
Good on mining the block before returning it (to be added to the block chain I presume) -- I forgot to do that I think.